include/runtime/instance/memory.h

```diff
@@ -261,16 +285,18 @@ class MemoryInstance {
     if (likely(Length > 0)) {
       if constexpr (std::is_floating_point_v<T>) {
         // Floating case. Do the memory copy.
-        std::memcpy(&Value, &DataPtr[Offset], sizeof(T));
+        *(reinterpret_cast<T *>(&Value)) =
+            *(reinterpret_cast<const T *>(&DataPtr[Offset]));
       } else {
         if constexpr (sizeof(T) > 8) {
-          assuming(sizeof(T) == 16);
+          static_assert(sizeof(T) == 16);
```

Comment:
Please use `assuming` here instead of `static_assert`. This will fit for both debug and release versions.

Reply:
Same reason to use static_assert as above.

Reply:
I have resumed to use the original `assume` instead. 

Thanks for pointing it out. I didn't know the if constexpr restriction.